plugins/services/src/js/service-configuration/ServiceHealthChecksConfigSection.js

```diff
@@ -6,12 +6,32 @@ import {
   getColumnHeadingFn,
   getDisplayValue
 } from '../utils/ServiceConfigDisplayUtil';
-import ConfigurationMapEditAction from '../components/ConfigurationMapEditAction';
 import ConfigurationMapDurationValue from '../components/ConfigurationMapDurationValue';
+import ConfigurationMapEditAction from '../components/ConfigurationMapEditAction';
+import ConfigurationMapHeading from '../../../../../src/js/components/ConfigurationMapHeading';
+import ConfigurationMapValue from '../../../../../src/js/components/ConfigurationMapValue';
 import ServiceConfigBaseSectionDisplay from './ServiceConfigBaseSectionDisplay';
 import {COMMAND, MESOS_HTTP, MESOS_HTTPS} from '../constants/HealthCheckProtocols';
 import Util from '../../../../../src/js/utils/Util';
 
+function renderDuration(prop, row) {
+  const value = row[prop];
+
+  if (!value || value === 0) {
+    return (
+      <ConfigurationMapValue>
+        {getDisplayValue(value)}
+      </ConfigurationMapValue>
+    );
+  }
+
+  return (
+    <ConfigurationMapDurationValue
```

Comment:
Perhaps we should fix this in the `ConfigurationMapDurationValue` instead / in addition?